servicecontrol/servicecontrol-instances/remotes.md

```diff
@@ -10,15 +10,15 @@ redirects:
 
 ## Overview
 
-One ServiceControl instance is designated as the _primary_ instance. All other ServiceControl instances are _remote_ instances. The HTTP API of the primary instance aggregates data from the primary instance and from all the remote instances. ServiceInsight and ServicePulse are configured to connect to the primary instance.
+One ServiceControl Error instance is designated as the _primary_ instance. All other ServiceControl instances are _remote_ instances. The HTTP API of the primary instance aggregates data from the primary instance and from all the remote instances. ServiceInsight and ServicePulse are configured to connect to the primary instance.
 
 NOTE: The term _remote_ refers to the fact that remote instances are run in separate processes. The primary instance and one or more remote instances can run on the same machine.
 
-In ServiceControl version 4 and later, a primary ServiceControl instance can be configured with remote instances that are [ServiceControl instances](/servicecontrol/servicecontrol-instances/) and [ServiceControl Audit instances](/servicecontrol/audit-instances/). ServiceControl Audit instances cannot be configured as primary instances.
+In ServiceControl version 4 and later, a ServiceControl Error instance can be configured with remote instances that are [ServiceControl instances](/servicecontrol/servicecontrol-instances/) and [ServiceControl Audit instances](/servicecontrol/audit-instances/). ServiceControl Audit instances cannot be configured as primary instances.
```

Comment:
Should this say this?

```suggestion
In ServiceControl version 4 and later, a ServiceControl Error instance can be configured with remote instances that are also [ServiceControl Error instances](/servicecontrol/servicecontrol-instances/) or are [ServiceControl Audit instances](/servicecontrol/audit-instances/). ServiceControl Audit instances cannot be configured as primary instances.
```

Also I noticed that the linked page (including it's name and menu entry) seem off:

<img width="677" alt="Screenshot 2023-07-28 at 12 55 23" src="https://github.com/Particular/docs.particular.net/assets/677704/a7b26f0b-f210-4d8a-b9eb-ef8a5ce86969">

Should that not all be "ServiceControl Error instances", `servicecontrol/servicecontrol-instances/`, etc?